### ISSUE TITLE
ovs_events: scan-build warnings

### ISSUE THREAD

**Issue by rubenk** [collectd/collectd#2205]
```
CC       src/ovs_events_la-utils_ovs.lo
src/utils_ovs.c:366:20: warning: Access to field 'type' results in a dereference of a null pointer (loaded from variable 'jval')
              (int)(jval)->type);
                   ^~~~~~~~~~~~
src/utils_ovs.c:93:32: note: expanded from macro 'OVS_ERROR'
    ERROR("ovs_utils: " fmt, ##__VA_ARGS__);                                   \
                               ^~~~~~~~~~~
./src/daemon/plugin.h:395:40: note: expanded from macro 'ERROR'
#define ERROR(...) plugin_log(LOG_ERR, __VA_ARGS__)
                                       ^~~~~~~~~~~
src/utils_ovs.c:570:9: warning: Null pointer argument in call to string comparison function
    if (strcmp("echo", method) == 0) {
        ^~~~~~~~~~~~~~~~~~~~~~
/usr/include/x86_64-linux-gnu/bits/string2.h:817:8: note: expanded from macro 'strcmp'
            : __builtin_strcmp (s1, s2)))); })
              ^~~~~~~~~~~~~~~~~~~~~~~~~
2 warnings generated.
  CCLD     ovs_events.la
  CC       src/ovs_stats_la-ovs_stats.lo
src/ovs_stats.c:353:15: warning: Access to field 'values' results in a dereference of a null pointer
              YAJL_GET_ARRAY(br_ports->u.array.values[1])->values;
              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/yajl/yajl_tree.h:180:28: note: expanded from macro 'YAJL_GET_ARRAY'
#define YAJL_GET_ARRAY(v)  (YAJL_IS_ARRAY(v)  ? &(v)->u.array  : NULL)
                           ^
src/ovs_stats.c:354:30: warning: Access to field 'len' results in a dereference of a null pointer
          size_t ports_num = YAJL_GET_ARRAY(br_ports->u.array.values[1])->len;
                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/yajl/yajl_tree.h:180:28: note: expanded from macro 'YAJL_GET_ARRAY'
#define YAJL_GET_ARRAY(v)  (YAJL_IS_ARRAY(v)  ? &(v)->u.array  : NULL)
                           ^
src/ovs_stats.c:540:38: warning: Access to field 'values' results in a dereference of a null pointer
      counter_name = YAJL_GET_STRING(YAJL_GET_ARRAY(stat)->values[0]);
                     ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/yajl/yajl_tree.h:180:28: note: expanded from macro 'YAJL_GET_ARRAY'
#define YAJL_GET_ARRAY(v)  (YAJL_IS_ARRAY(v)  ? &(v)->u.array  : NULL)
                           ^
/usr/include/yajl/yajl_tree.h:162:44: note: expanded from macro 'YAJL_GET_STRING'
#define YAJL_GET_STRING(v) (YAJL_IS_STRING(v) ? (v)->u.string : NULL)
                                           ^
/usr/include/yajl/yajl_tree.h:150:30: note: expanded from macro 'YAJL_IS_STRING'
#define YAJL_IS_STRING(v) (((v) != NULL) && ((v)->type == yajl_t_string))
                             ^
src/ovs_stats.c:560:29: warning: Access to field 'values' results in a dereference of a null pointer
      key = YAJL_GET_STRING(YAJL_GET_ARRAY(ext_id)->values[0]);
            ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/yajl/yajl_tree.h:180:28: note: expanded from macro 'YAJL_GET_ARRAY'
#define YAJL_GET_ARRAY(v)  (YAJL_IS_ARRAY(v)  ? &(v)->u.array  : NULL)
                           ^
/usr/include/yajl/yajl_tree.h:162:44: note: expanded from macro 'YAJL_GET_STRING'
#define YAJL_GET_STRING(v) (YAJL_IS_STRING(v) ? (v)->u.string : NULL)
                                           ^
/usr/include/yajl/yajl_tree.h:150:30: note: expanded from macro 'YAJL_IS_STRING'
#define YAJL_IS_STRING(v) (((v) != NULL) && ((v)->type == yajl_t_string))
                             ^
4 warnings generated.
  CC       src/ovs_stats_la-utils_ovs.lo
src/utils_ovs.c:366:20: warning: Access to field 'type' results in a dereference of a null pointer (loaded from variable 'jval')
              (int)(jval)->type);
                   ^~~~~~~~~~~~
src/utils_ovs.c:93:32: note: expanded from macro 'OVS_ERROR'
    ERROR("ovs_utils: " fmt, ##__VA_ARGS__);                                   \
                               ^~~~~~~~~~~
./src/daemon/plugin.h:395:40: note: expanded from macro 'ERROR'
#define ERROR(...) plugin_log(LOG_ERR, __VA_ARGS__)
                                       ^~~~~~~~~~~
src/utils_ovs.c:570:9: warning: Null pointer argument in call to string comparison function
    if (strcmp("echo", method) == 0) {
        ^~~~~~~~~~~~~~~~~~~~~~
/usr/include/x86_64-linux-gnu/bits/string2.h:817:8: note: expanded from macro 'strcmp'
            : __builtin_strcmp (s1, s2)))); })
              ^~~~~~~~~~~~~~~~~~~~~~~~~
2 warnings generated.
  CCLD     ovs_stats.la
```

**Comment by rubenk** [comment]
The build log is at https://ci.collectd.org/job/master-build-on-xenial-amd64-with-scan-build/322/console

@maryamtahhan please have a look.

**Comment by maryamtahhan** [comment]
Hi @rubenk 
@vmytnykx  is going to look into this

**Comment by rubenk** [comment]
Fixed by 4256753373b8f61dc1928dfc7b0564f2fbc32289
